sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -1293,10 +1298,6 @@ def _generate_delete_blobs_subrequest_options(
         modified_access_conditions=None,
         **kwargs
     ):
-        """This code is a copy from _generated.
-
-        Once Autorest is able to provide request preparation this code should be removed.
```

Comment:
Can you turn this into just a regular comment. I think its good to note this but we don't need it in a docstring.

Reply:
Good catch. I remember nuking it, then in other files I turned it into just a regular comment but then I couldn't find the nuked sites. Will do!